tests/integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -128,16 +129,25 @@ fi
 
 ensure_yq
 
-info "Running tests with bats version: $(bats --version)"
+report_dir="${kubernetes_dir}/reports/$(date +'%F-%T')"
+mkdir -p "$report_dir"
+
+info "Running tests with bats version: $(bats --version). Save outputs to $report_dir"
 
 tests_fail=()
 for K8S_TEST_ENTRY in ${K8S_TEST_UNION[@]}
 do
 	info "$(kubectl get pods --all-namespaces 2>&1)"
 	info "Executing ${K8S_TEST_ENTRY}"
-	if ! bats --show-output-of-passing-tests "${K8S_TEST_ENTRY}"; then
+	# Output file will be prefixed with either "ok" or "not_ok" in case of
+	# success or fail, respectively.
+	out_file="${report_dir}/${K8S_TEST_ENTRY}.out"
+	if ! bats --show-output-of-passing-tests "${K8S_TEST_ENTRY}" | tee "$out_file"; then
```

Comment:
Provided we start logging the output of tests later it might make sense to avoid printing it here to avoid unnecessary logs, what do you think? Also shouldn't we capture the stderr as well to then replay it with the bats output?